ui/component/fileDetails/view.jsx

```diff
@@ -108,7 +90,13 @@ class FileDetails extends PureComponent<Props> {
               )}
             </tbody>
           </table>
-        </Expandable>
+        </ExpandingContainer>
```

Comment:
@seanyesmunt consider if these should just be expandable/collapsable cards or otherwise please give light styling love

Reply:
PR for expandable cards on the change shelf